src/effects/aura.c

```diff
@@ -1,11 +1,194 @@
 #include "common.h"
 #include "effects_internal.h"
 
+extern Gfx D_090023E0[];
+extern Gfx D_09002440[];
+extern Gfx D_090023F8[];
+extern Gfx D_09002458[];
+extern Gfx D_090023B0[];
+extern Gfx D_09002410[];
+extern Gfx D_090023C8[];
+extern Gfx D_09002428[];
+extern Gfx D_09002000[];
+extern Gfx D_090020E8[];
+extern Gfx D_090021D0[];
+
+Gfx* D_E0076E90[] = { D_090023E0, D_090023E0, D_09002440, D_09002440 };
+Gfx* D_E0076EA0[] = { D_090023F8, D_090023F8, D_09002458, D_09002458 };
+Gfx* D_E0076EB0[] = { D_090023B0, D_090023B0, D_09002410, D_09002410 };
+Gfx* D_E0076EC0[] = { D_090023C8, D_090023C8, D_09002428, D_09002428 };
+Gfx* D_E0076ED0[] = { D_09002000, D_09002000, D_090020E8, D_090021D0 };
+
 void aura_appendGfx(void* effect);
 
-INCLUDE_ASM(s32, "effects/aura", aura_main);
+typedef struct AuraFXData {
```

Comment:
Effect data structs should go into effects.h (in order, which is kinda annoying atm, sorry for the inconvenience)

Reply:
also relevant for your other structs below